src/test/kotlin/org/opensearch/replication/bwc/BackwardsCompatibilityIT.kt

```diff
@@ -139,6 +142,19 @@ class BackwardsCompatibilityIT : MultiClusterRestTestCase() {
                 Assertions.assertThat(getResponse.isExists).isTrue()
                 Assertions.assertThat(getResponse.sourceAsMap).isEqualTo(source)
             }, 60, TimeUnit.SECONDS)
+
+            //Check for latest retention lease when full cluster restart is done
```

Comment:
why didn't we add testcase for rolling restart?

Reply:
Added